ansible_base/feature_flags/views.py

```diff
@@ -0,0 +1,24 @@
+from django.utils.translation import gettext_lazy as _
+from rest_framework.response import Response
+
+from ansible_base.feature_flags.serializers import FeatureFlagSerializer
+from ansible_base.lib.utils.views.ansible_base import AnsibleBaseView
+
+from .utils import get_django_flags
+
+
+class FeatureFlagsListView(AnsibleBaseView):
+    """
+    A view class for displaying feature flags
+    """
+
+    serializer_class = FeatureFlagSerializer
+    filter_backends = []
+    name = _('Feature Flags')
+    http_method_names = ['get', 'head']
+
+    def get(self, request, format=None):
+        return Response(get_django_flags())
+
+    def get_queryset(self):
+        return get_django_flags()
```

Comment:
What permissions should be set on this view strikes me as a shared problem with `/api/v2/ping/` in AWX, as well as others. This is what https://github.com/ansible/awx/pull/15724 was seeking to address. Doing a pure-AWX solution is not the right answer here, but I don't have a good answer as to what the answer should be.

Reply:
Hey @AlanCoding ,
Is there anything that I can do to better represent this now? Sounds like we may want to open another issue to address this later?

Reply:
I'm assuming the default and expectation would be that all authenticated users can view, but I'd first ask the PM driving this. Otherwise, no, there's nothing better I could do.

Reply:
This API endpoint is only valid for authenticated users currently. Resolving this convo, but please feel free to reopen if you think this should be discussed more